lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -57,6 +57,8 @@ async function runA11yChecks() {
       'svg-img-alt': {enabled: false},
       'audio-caption': {enabled: false},
       'aria-treeitem-name': {enabled: true},
+      'empty-table-header': {enabled: true},
+      'aria-text': {enabled: true},
```

Comment:
adding these for a reason if they're not in `wcag2a`/`wcag2aa`?

Reply:
Honest answer is I did not realize we start with those as a base `runOnly` and tweak the rest. I just added all the new rules and tweaked this section until results showed up in the sample LHR. Some comments or docs here explaining our criteria for inclusion would help–although even now I don't know what that'd be, since there are some items here that are manually turned on.